server/@types/bapv.d.ts

```diff
@@ -176,7 +176,7 @@ export type VisitSessionData = {
     location: string
   }
   visit?: VisitSlot
-  visitRestriction?: 'OPEN' | 'CLOSED'
+  visitRestriction?: 'OPEN' | 'CLOSED' | 'UNKNOWN'
```

Comment:
I guess the `UNKNOWN` is needed to show migrated visits that may not have a type set?

Reply:
It was moaning at me that it was in the pulled in types.